### DISCUSSION TITLE
Is Home Assistant or Homekit Integration Possible?

### DISCUSSION THREAD

**Discussion by hackdefendr** [spensbot/captivate#22] [Ideas]
Being into the whole home automation thing and also an amature EDM home producer, Captivate has my attention. I fully plan on looking at the code and seeing if I can contribute, but wanted to see what the community thinks. Personally when I am making beats I like the mood to be right and automated colored lighting does just that.

Could Captivate be somehow used with Home Assistant or Homekit to do some wild lighting scenes?  

Now Home Assistant does have a couple of DMX integrations, so the thought was to make a plugin for Captivate to send scenes to those integrations so I can create that club like experience using my home lighting along side the strobes I have when I'm feeling musical.

First, is it possible and worth exploring?

Peace,
Jeff

**Comment by stonegray** [comment]
Home assistant doesn't like rapid data updates, as every hue/brightness change is logged and send in a websocket to every connected web and app client, even if they don't display that data. 

You could probably bridge the ArtNet over but I wouldn't recommend it.

If you integrated directly with your bulbs with a way to suppress updates it might be possible.
